code/mob/living/critter/ai/mouse.dm

```diff
@@ -0,0 +1,25 @@
+/datum/aiHolder/mouse
+	New()
+		..()
+		default_task = get_instance(/datum/aiTask/prioritizer/critter/mouse, list(src))
+
+/datum/aiTask/prioritizer/critter/mouse/New()
+	..()
+	transition_tasks += holder.get_instance(/datum/aiTask/timed/wander, list(holder, src))
+	transition_tasks += holder.get_instance(/datum/aiTask/sequence/goalbased/critter/eat, list(holder, src))
+
+/datum/aiHolder/mouse_mad
```

Comment:
I kind of want this to be `/datum/aiHolder/mouse/mad`, ditto for the prioritizer. It doesn't matter, but it is a little cleaner.